.changeset/pretty-yaks-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/ui-extensions': patch
+---
+
+udpate readme to reflect 2025-10 related api changes
```

Comment:
```suggestion
Update readme to reflect 2025-10 related api changes
```

Reply:
well I think without a changset it won't trigger a release so the readme will remain out of date, right? or do you think the readme would update regardless?